### PR TITLE
db setup

### PR DESCRIPTION
**Pull request by Jovialiste82**
Finished DB setup.
Can you please review ?

### CODE REVIEW

File: server/.env [hunk 1]
```diff
@@ -1,2 +1,4 @@
 TEAM_NAME="Jason Chui, Philippe Charpentier, Ziqi Shen, Milan Verma"
 FLASK_ENV="development"
+MONGODB_PWD="replace with DBAdminPWD provided on slack"
+TEST_ENV="Hello Team BLT"
```
Comment (jessdesr) [inline server/.env:4]: This file can be added to the .gitignore so it no longer shows up in the repository 👍

File: server/app.py [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+import pymongo
+import dns
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
 
-
 app = Flask(__name__)
 
+####### Added by Philippe ######### start
+######### DB SETUP ######################
```
Comment (jessdesr) [inline server/app.py]: No need for these comments

File: server/app.py [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+try:
+  # client = pymongo.MongoClient("mongodb+srv://TeamBLT-Admin:<password>@teambltcluster0.5hi8c.mongodb.net/<dbname>?retryWrites=true&w=majority")
```
Comment (jessdesr) [inline server/app.py]: clear out any commented out code as well

**Review by jessdesr — APPROVED** [review]
Approved, just make the changes I've suggested and it can be merged

**Comment by Jovialiste82** [comment]
Thx Jess. I just cleared out all the comments and committed again. Merging now. Will then pull to my local machine and add .env to gitignore

**Pull request merged**